config/labels.yml

```diff
@@ -250,6 +250,10 @@ repos:
     <<: *repo_introduced_in_ivanchuk
     <<: *provider_plugin
     <<: *release
+  ManageIQ/manageiq-providers-cisco_intersight:
```

Comment:
NOTE I did not include any `repo_introduced_in_*` as this is not in any release branch yet

Reply:
:+1: will do